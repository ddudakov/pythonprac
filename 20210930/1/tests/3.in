1234, 5, 6, 1, 2, 54221
