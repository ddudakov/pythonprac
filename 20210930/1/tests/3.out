[1, 2, 5, 6, 54221, 1234]
